scalableminds/webknossos
# Add default organization to onboarding flow

Added a default organization that is used in the onboarding flow. The organization is set in `application.conf -> features`. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Log out and open `<base_url>/onboarding`.
- Now you can select an organization. The default organization should be the one that is defined in `application.conf`.

### Issues:
- fixes #3970

------
